app/models/assignment_team.rb

```diff
@@ -42,43 +62,52 @@ def topic
 
   # Whether the team has submitted work or not
   def has_submissions?
-    list_of_users = participants;
-    list_of_users.each { |participant| return true if participant.has_submissions? }
-    false
+    (self.submitted_files.length > 0) or self.submitted_hyperlinks.blank?
   end
 
+  #Response Map for each reviewer/reviewee
   def reviewed_contributor?(contributor)
+    #ReviewResponseMap.all showing error but somehow its working, incase it fails please uncomment the following line
```

Comment:
I cannot find any place that this method is called? Could you delete this if you can confirm that is it not called anywhere?